package.json

```diff
@@ -37,30 +37,35 @@
     "organization": false
   },
   "devDependencies": {
-    "@monocdk-experiment/assert": "^1.123.0",
+    "@monocdk-experiment/assert": "1.123.0",
     "@types/jest": "^27.4.1",
     "@types/node": "^12",
     "@typescript-eslint/eslint-plugin": "^5",
     "@typescript-eslint/parser": "^5",
+    "constructs": "3.3.69",
     "eslint": "^8",
+    "eslint-config-prettier": "^8.4.0",
     "eslint-import-resolver-node": "^0.3.6",
     "eslint-import-resolver-typescript": "^2.5.0",
     "eslint-plugin-import": "^2.25.4",
+    "eslint-plugin-prettier": "^4.0.0",
     "jest": "^27.5.1",
     "jest-junit": "^13",
     "jsii": "^1.54.0",
     "jsii-diff": "^1.54.0",
     "jsii-docgen": "^6.1.12",
     "json-schema": "^0.4.0",
+    "monocdk": "1.123.0",
     "npm-check-updates": "^12",
+    "prettier": "^2.5.1",
     "projen": "^0.52.44",
     "standard-version": "^9",
     "ts-jest": "^27.1.3",
     "typescript": "^4.5.5"
   },
   "peerDependencies": {
     "constructs": "^3.2.27",
-    "monocdk": "^1.123.0"
+    "monocdk": "1.123.0"
```

Comment:
This shouldn't have changed, should it?

Reply:
Now ti should be fine, I added some more manual dep changes.